docs/UserGuide.md

```diff
@@ -53,7 +53,11 @@ With NUScheduler, you can store the contact details of the NUS community around
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
   e.g. `pr/PROFILE_INDEX...` must be used at least once, such as `pr/1`, `pr/3 pr/5` etc.
 
-* For Event and Profile commands, option flags (-x where x is a placeholder for valid flags e.g. a, d, v) must be used after the event and profile command word.
+* For `Event` and `Profile` commands, option flags (-x where x is a placeholder for valid flags e.g. a, d, v) must be used after the `event` and `profile` command word.
+
+* For `Event` and `Profile` commands which require an `Index`, the `Index` must be provided after the option flag. <br>
+
+* In addition, the `Index` provided must be a positive number less than 10000 as `NUScheduler` is designed to handle a maximum of 9999 entries for `Profile`s and `Event`s which covers the needs for most NUS students.
```

Comment:
Perhaps we don't need the back ticks for NUScheduler to keep it in line with the formatting of NUScheduler throughout our user guide.